lib/stm32/f7/Makefile

```diff
@@ -26,7 +26,7 @@ PREFIX		?= arm-none-eabi
 CC		= $(PREFIX)-gcc
 AR		= $(PREFIX)-ar
 
-# STM32F7 only supports single precision FPU
+# Some STM32F7 only support single precision FPU
```

Comment:
out of scope of this commit, but yes, definitely true :)

Reply:
ah crap.. yeah, i do those fly-by changes when i am tired